boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java

```diff
@@ -86,7 +86,7 @@ private void createSpringBootDockerFile(String springBootVersion) throws MojoExe
     }
     
     @SuppressWarnings("resource")
-    private boolean isFileExecutable(File file) throws IOException {
+    private boolean isFileExecutable(File file) throws IOException, MojoExecutionException {
         if (file.exists()) {
```

Comment:
Add more checks for the file, i.e. `if(file != null && file.exists() && file.isFile())`

Reply:
fixed